src/test/java/seedu/address/logic/parser/TaTrackerParserTest.java

```diff
@@ -16,22 +16,21 @@
 import seedu.address.logic.commands.AddCommand;
 import seedu.address.logic.commands.ClearCommand;
 import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
```

Comment:
missing replacement for this, need to include `import seedu.address.logic.commands.EditStudentCommand.EditStudentDescriptor`